.github/workflows/windows-meson-msys2.yml

```diff
@@ -75,26 +242,51 @@ jobs:
       - name: Strip binary
         run: strip build/dosbox.exe
       
-      - name: Package
+      - name: Stage release binary
         run: |
           set -x
-          mkdir -p dest/doc
+          mkdir -p ${{ env.PKG_DEST_DIR }}/
+          cp build/dosbox.exe ${{ env.PKG_DEST_DIR }}/
+          ntldd -R ${{ env.PKG_DEST_DIR }}/dosbox.exe \
+            | sed -e 's/^[[:blank:]]*//g' \
+            | cut -d ' ' -f 3 \
+            | grep -E -i '(mingw|clang)(32|64)' \
+            | sed -e 's|\\|/|g' \
+            | xargs cp --target-directory=${{ env.PKG_DEST_DIR }}/
```

Comment:
It would be nice to avoid repeating this shell block in two places.
Ideally we could make this a shell script that takes in two arguments:
 - Destination directory
 - Path to the source executable

The script would do the work done here.. 

We've got the top-level `./scripts/` path and also `.github/scripts` (for things that are github-centric).

For users building on MSYS2, this would be very handy for them to "lift out" a package into a nice stand-alone directory.

That said - these are only wishes; you've done a ton of work here, and it all works - so we would consider this work as a 'rev2' polishing step.  I'm happy with either approach!

Reply:
I agree. Don't really like the repetition myself. The other option is to make the debugger builds a separate job in the matrix.

Reply:
> The other option is to make the debugger builds a separate job in the matrix.

Because the ccache'd builds only take 1/20th time as it does to stand up and configure an MSYS2 VM,  we're gaining a lot of efficiency by completing multiple builds in the same VM instance.  So would prefer we stick to this approach, if possible.

Reply:
I've made some changes to avoid repetition. Looking forward to your feedback on the changes.